src/opensles/AudioStreamBuffered.h

```diff
@@ -64,7 +64,7 @@ class AudioStreamBuffered : public AudioStream {
     // If there is no callback then we need a FIFO between the App and OpenSL ES.
     bool usingFIFO() const { return getCallback() == nullptr; }
 
-    virtual Result updateServiceFrameCounter() { return Result::OK; }
+    virtual Result updateServiceFrameCounter() = 0;
```

Comment:
How did you know that this could be made a pure virtual? Do you have a specific compiler warning on?

Reply:
I tried it, following a comment in #352. Building a project that uses Oboe still worked. I think making this pure virtual also makes AudioStreamOpenSLES pure virtual, but it is never instantiated, and only the Input/Ouput subclasses are.